packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -691,9 +732,12 @@ const onSelection = (id: string) => {
 
 const fetchConfigurations = async (modelId: string) => {
 	if (modelId) {
-		loadingConfigs.value = true;
-		suggestedConfirgurationContext.value.tableData = await getModelConfigurations(modelId);
-		loadingConfigs.value = false;
+		// FIXME: since configurations are made on the backend on the fly, we need to wait for the db to update before fetching, here's an artificaial delay
```

Comment:
Off the top of my head I don't have a _better_ solution that isn't adding a bunch of work - polling, mq, etc - but this is something that should for sure at minimum have a high priority issue opened up today to address as soon as the hackathon is over! At best the user is going to have to wait an extra ~second here, but at worse there isn't a guarantee that 800ms would be long enough then we're back to where we were before.

Reply:
@kbirk is looking into adding a way to get a finished message after BE is done processing